test/conftest.py

```diff
@@ -28,3 +30,89 @@ def config():
     with open(path_config, "r") as file:
         config_dict = yaml.safe_load(file)
     return config_dict
+
+
+@pytest.fixture(scope="function")
```

Comment:
Hey @lkstrp,

indeed great question! 

I usually used _function_ being the default one and (most likely) the safest one.

However, now that I think about it, the `config` fixture can be initialized only once per test session. Changing the scope for this fixture actually leads to a speed up of the test execution (you can find the results below).

This is of course not very relevant now, but it will become once we add more and more tests.

```
@pytest.fixture(scope="session")
def config():

hyperfine pytest test

Time (mean ± σ):      1.568 s ±  0.004 s    [User: 1.333 s, System: 0.228 s]
Range (min … max):    1.563 s …  1.573 s    10 runs

```